WooCommerce/WooCommerceTests/Authentication/Jetpack Setup/LoginJetpackSetupViewModelTests.swift

```diff
@@ -0,0 +1,285 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+
+final class LoginJetpackSetupViewModelTests: XCTestCase {
+    private let testURL = "https://test.com"
+
+    func test_title_is_correct_if_jetpack_installation_is_required() {
+        // Given
+        let viewModel = LoginJetpackSetupViewModel(siteURL: testURL, connectionOnly: false)
+
+        // Then
+        XCTAssertEqual(viewModel.title, LoginJetpackSetupViewModel.Localization.installingJetpack)
```

Comment:
Nice work using the strings from the view model directly. We don't have to duplicate the strings here again. I will use this technique moving forward.